LocalMultiplayerAgent/SettingsJsonGenerator/index.js

```diff
@@ -1 +1 @@
details={Registry:t,ImageName:n,ImageTag:r,Username:o,Password:i}}m(h("MountPath"),"MountPath",e.AssetDetails[0]),m(h("GamePortNumber"),"Number",e.PortMappingsList[0][0].GamePort)}m(h("OutputFolder"),"OutputFolder",e),m(h("LocalFilePath"),"LocalFilePath",e.AssetDetails[0]),m(h("NumHeartBeatsForActivateResponse"),"NumHeartBeatsForActivateResponse",e),m(h("NumHeartBeatsForTerminateResponse"),"NumHeartBeatsForTerminateResponse",e),m(h("AgentListeningPort"),"AgentListeningPort",e),m(h("NodePort"),"NodePort",e.PortMappingsList[0][0]),m(h("GamePortName"),"Name",e.PortMappingsList[0][0].GamePort),m(h("GamePortProtocol"),"Protocol",e.PortMappingsList[0][0].GamePort),document.getElementById("outputText").value=JSON.stringify(e,null,2),function(){let e=h("RunMode"),t=h("StartCommand"),n="";if(e===c)-1===t.search(":")||(n="The Start Command should be a relative path into the zip file, not an absolute path.");else if(e===f){let e=h("MountPath");0===t.search(e)||(n="The Start Command should be an absolute path that starts with the Asset Mount Path")}else e===p&&(h("MountPath"),t&&(n="The Start Command should be empty (The container should launch the GSDK and Game Server directly)"));document.getElementById("linux-docker-fields").className=e===p?"":"hidden",document.getElementById("StartCommandValidate").innerHTML=n}(),function(){let e="";-1!=h("LocalFilePath").search("fakepath")&&(e="Warning: This browser obscures the actual path of files. You will need to manually fix the LocalFilePath in the json"),document.getElementById("LocalFilePathValidate").innerHTML=e}(),function(){let e="",t="",n=h("RunMode"),r=h("MountPath");n===c?0!=r.search(l)&&(t=d):n===f?0!=r.search("C:\\\\")?e="Your path must start with the C:\\ drive for Windows containers":0!=r.search(l)&&(t=d):n===p&&0!=r.search("/Data/")&&(t="We recommended you choose a sub-folder of /Data"),document.getElementById("MountPathValidate").innerHTML=e,document.getElementById("MountPathWarning").innerHTML=t}()}const g=Array.prototype.slice.call(document.getElementsByTagName("input"),0),y=Array.prototype.slice.call(document.getElementsByTagName("select"),0);g.forEach((e=>{e.addEventListener("change",(()=>{const t=!i(e.getAttribute("min")),n=!i(e.getAttribute("max"));if(!isNaN(Number(e.value))&&(t||n)){let r=Number(e.value);if(t){const t=Number(e.getAttribute("min"));r<t&&(r=t)}if(n){const t=Number(e.getAttribute("max"));r>t&&(r=t)}e.value=r}v()}))})),y.forEach((e=>{e.addEventListener("change",(()=>{v()}))})),v(),(0,o.default)("[data-tippy-content]")}},t={};function n(r){var o=t[r];if(void 0!==o)return o.exports;var i=t[r]={exports:{}};return e[r].call(i.exports,i,i.exports,n),i.exports}n.d=(e,t)=>{for(var r in t)n.o(t,r)&&!n.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},n.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),n.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n(607)})();
```

Comment:
>(()=>{" [](http://example.com/codeflow?start=0&length=7)

I can't see any changes in this 'SettingsJsonGenerator' folder, should it be excluded from the PR?